src/Phone.java

```diff
@@ -0,0 +1,15 @@
+public interface Phone {
```

Comment:
since phone can also store private members like battery life, an abstract class would be more suitable. also, you could implement here functionality that is common to all phones